src/Angular.js

```diff
@@ -143,6 +144,7 @@ var minErrConfig = {
  * current configuration if used as a getter. The following options are supported:
  *
  * - **objectMaxDepth**: The maximum depth to which objects are traversed when stringified for error messages.
+ * - **urlMaxLength**: The maximum length of the URL reference in the error messages.
```

Comment:
Limiting the array length makes sense (although not directly related to this PR).